databricks/provider_test.go

```diff
@@ -107,12 +107,40 @@ func TestProvider(t *testing.T) {
 	}
 }
 
-func TestProvider_NoOptionsResultsInError(t *testing.T) {
+func TestProvider_NoOptionsDoesNotResultsInError(t *testing.T) {
```

Comment:
@EliiseS no provider configuration should load data from `~/.databrickscfg` and error on it's absence - same behaviour as `azurerm` and `aws` providers.